hws/src/dr_common.py

```diff
@@ -178,9 +178,9 @@ def call_resource_dump(dev, dev_name, segment, index1, num_of_obj1, num_of_obj2,
     if vhca_id != None and vhca_id != "0":
         _input += ' --virtual-hca-id ' + vhca_id
 
-    output = sp.getoutput(_input)
-
-    if (len(output) >= 10) and ('Error' in output[0:10]):
+    status, output = sp.getstatusoutput(_input)
+    # FIXME: "Error" isn't always printed on the first line.
```

Comment:
correct,
remove the FIXME line

Reply:
The current code doesn't detect "Error" when it's not printed on the first line, hence there's an outstanding bug. Shouldn't we leave the FIXME message in the code so we remember to fix this issue sometime? Or now that the code checks the process exit code, should we just completely remove the logic related to the "Error" string?

Reply:
we can keep both ways exit code and error, as i do not know if they always return correct exit code in case of an error, as this was the case we used the "error".

maybe the MFT tools more stable now and we can just depend on the status code

Reply:
OK, I'll leave the "Error" check in place since I am not familiar enough with the internals of MFT to know if the exit code is 100% reliable. I think we should leave the FIXME comment in place since the "Error" checking code is incomplete at present.

So I think this pull request is ready to merge now.